resolwe_bio/processes/support_processors/mutations_table.py

```diff
@@ -240,6 +241,39 @@ def get_depth(variants_table, bam):
     return variants_table.to_csv("mutations.tsv", sep="\t")
 
 
+def encode_variant_genotype(row):
+    """Encode variant to genotype notation.
+
+    Variants are given as <allele1>/<allele2>, e.g. T/T or C/T
+
+    Encode these variants as:
+        - 0/1 for heterozygous variants
+        - 1/1 for homozygous variants
+    """
+    try:
+        allele_line = row.get("genotype", np.nan)
+        allele_re = r"^([ACGT]+)/([ACGT]+)$"
```

Comment:
I am aware this is an existing regex used by resdk but it fails to capture some of the variants. In particular `*/G` and similarly structured ones. We could add in the asterisk in the pattern `"^([ATGC*]+)/([ATGC*]+)$"`.

Reply:
Fixed as suggested, thanks!